src/main/java/org/sonar/plugins/stash/StashRequestFacade.java

```diff
@@ -185,8 +185,18 @@ public void postCommentPerIssue(String project, String repository, String pullRe
     }
   }
   
-  public StashCredentials getCredentials(){
-    return new StashCredentials(config.getStashLogin(), config.getStashPassword());
+  public StashCredentials getCredentials() throws StashConfigurationException {
```

Comment:
Hm, it should not matter. If they configured the environment and did not use it, the login should fail whether due to an exception by us or an invalid login to the stash server would not matter.